src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3179,6 +3179,12 @@ private static bool IsNegativeConstantForArraySize(BoundExpression expression)
 
         protected BoundExpression BindObjectCreationExpression(ObjectCreationExpressionSyntax node, DiagnosticBag diagnostics)
         {
+            if (node.Type.Kind() == SyntaxKind.TupleType)
+            {
+                Debug.Assert(node.HasErrors, "new <tuple type> should be a syntax error");
+                return BadExpression(node, LookupResultKind.NotCreatable);
```

Comment:
Agreed. If we happened to parse it (not make a junk trivia), we may as well bind it. 
Perhaps should put "HasError" flah on the bound node though. To make sure it does not go further.

---
In reply to: [81652702](https://github.com/dotnet/roslyn/pull/14161#discussion_r81652702) [](ancestors = 81652702)